casper/src/main/rholang/NonNegativeNumber.rho

```diff
@@ -0,0 +1,42 @@
+//scalapackage coop.rchain.casper.genesis.contracts
+
+//This is a contract to explicitly implement the int >= 0 behavioural type
+contract @"NonNegativeNumber"(@init, return) = {
+  new this, valueStore in {
+    contract @[*this, "add"](@x, success) = {
+      match x >= 0 {
+        true => {
+          for(@v <- valueStore){
+            valueStore!(v + x) | success!(true)
+          }
+        }
+        _ => { success!(false)  }
+      }
+    } |
+    contract @[*this, "sub"](@x, success) = {
+      match x >= 0 {
```

Comment:
Using if-then-else instead of match-true-false would be more readable

Reply:
Yes, here and everywhere else.